src/components/Api.js

```diff
@@ -25,10 +46,15 @@ export default class Api {
     });
```

Comment:
` .then(this._checkResponse)` should be everywhere: after `deleteCard` and `dislikeCard` as well